src/components/now-playing-drawer/Artwork.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+
+import { useTrackCoverArt } from 'audius-client/src/common/hooks/useImageSize'
+import { SquareSizes } from 'audius-client/src/common/models/ImageSizes'
+import { Track } from 'audius-client/src/common/models/Track'
+import { getDominantColorsByTrack } from 'audius-client/src/common/store/average-color/slice'
+import { Dimensions, StyleSheet, View } from 'react-native'
+import { Shadow } from 'react-native-shadow-2'
+
+import DynamicImage from 'app/components/dynamic-image'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+const dimensions = Dimensions.get('window')
+const spacing = 24
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    root: {
+      marginLeft: spacing,
+      marginRight: spacing,
+      width: dimensions.width - spacing * 2,
+      height: dimensions.width - spacing * 2
+    },
+    shadow: {
+      alignSelf: 'stretch',
+      height: '100%'
+    },
+    image: {
+      borderRadius: 8,
+      borderColor: themeColors.white,
+      borderWidth: 2,
+      overflow: 'hidden',
+      height: '100%',
+      width: '100%'
+    }
+  })
+
+type ArtworkProps = {
+  track: Track | null
+}
+
+export const Artwork = ({ track }: ArtworkProps) => {
+  const styles = useThemedStyles(createStyles)
+  const image = useTrackCoverArt(
+    track?.track_id ?? null,
```

Comment:
is it okay for trackCoverArt to accept undefined, so you can just do `track?.track_id` etc?

Reply:
unf useTrackCoverArt needs nulls instead of undefines. can look into this change separately!